Mathlib/CategoryTheory/Products/Basic.lean

```diff
@@ -313,6 +313,32 @@ def compEvaluation (F : A ⥤ B ⥤ C) (b) : F ⋙ (evaluation _ _).obj b ≅ F.
 theorem comp_evaluation (F : A ⥤ B ⥤ C) (b) : F ⋙ (evaluation _ _).obj b = F.flip.obj b :=
   rfl
 
+/-- Whiskering by `F` and then evaluating at `a` is the same as evaluating at `F.obj a`. -/
+@[simps!]
+def whiskeringLeftCompEvaluation (F : A ⥤ B) (a : A) :
+    (whiskeringLeft A B C).obj F ⋙ (evaluation A C).obj a ≅ (evaluation B C).obj (F.obj a) :=
+  NatIso.ofComponents (fun _ => Iso.refl _)
```

Comment:
Why not just `Iso.refl _`?

Reply:
The reason was that once you unfold `whiskeringLeftCompEvaluation.hom` to an identity, you might end up with goals that cannot be solved anymore without `erw`. But honestly unfolding to `NatIso.ofComponents` isn't much better; the real solution is to only use the lemmas generated by `simps!` and those are the same either way. So I'll change it to `Iso.refl _`.